### PR TITLE
chore(scripts): tweak script run docs

### PR DESCRIPTION
**Pull request by snakster**
## What this PR does / why we need it:

This is a rewording of the `script run` docs to be more clear about command labels.

## Does this PR introduce a user-facing change?

```
no
```

### CODE REVIEW

**Review by wmalik — DISMISSED** [review]
Looks much better

File: docs/cli/cmdline/script/run.md [hunk 1]
```diff
@@ -7,24 +7,33 @@ description: With the terramate
...
 - any filters match. `script run` currently supports `--changed` and `--tags` filters.
 
+It's also possible to define commands that consist of multiple labels:
+
+```
+script "label1" "label2" ... {
+  ...
+}
+```
+The above can be run with `script run label1 label2`.
```
Comment (sixstone-qq) [inline docs/cli/cmdline/script/run.md:32]: Is this an `or` or an `and`? I mean, would it run if the argument is `script run label1`?

Comment (snakster) [reply docs/cli/cmdline/script/run.md:32]: This is an and, and the sequence needs to match. So basically just syntactic sugar for allowing whitespaces in commands.

Comment (snakster) [reply docs/cli/cmdline/script/run.md]: I refined the wording, wdyt?

**Review by sixstone-qq — APPROVED** [review]
Lgtm

**Review by i4ki — APPROVED** [review]
LGTM

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Pull request merged**